Controller/TutorialController.php

```diff
@@ -0,0 +1,221 @@
+                    'required' => false,
+                )
+            )
+            ->add(
+                'date_example8',
+                'date',
+                array(
+                    'widget' => 'single_text',
+                    'datepicker' => true,
+                    'date_type' => 'range',
+                    'minimum_date' => new \DateTime('last monday'),
+                    'maximum_date' => new \DateTime('next friday'),
+                    'required' => false,
+                )
+            )
+            ->add(
+                'datetime_example1',
+                'datetime',
+                array(
+                    'widget' => 'choice',
+                    'required' => false,
+                )
+            )
+            ->add(
+                'datetime_example2',
+                'datetime',
+                array(
+                    'widget' => 'text',
+                    'required' => false,
+                )
+            )
+            ->add(
+                'datetime_example3',
+                'datetime',
+                array(
+                    'widget' => 'single_text',
+                    'required' => false,
+                )
+            )
+            ->add(
+                'time_example1',
+                'time',
+                array(
+                    'widget' => 'choice',
+                    'required' => false,
+                )
+            )
+            ->add(
+                'time_example2',
+                'time',
+                array(
+                    'widget' => 'text',
+                    'required' => false,
+                )
+            )
+            ->add(
+                'time_example3',
+                'time',
+                array(
+                    'widget' => 'single_text',
+                    'required' => false,
+                )
+            )
+            ->add(
+                'birthday_example1',
+                'birthday',
+                array(
+                    'widget' => 'choice',
+                    'required' => false,
+                )
+            )
+            ->add(
+                'birthday_example2',
+                'birthday',
+                array(
+                    'widget' => 'text',
+                    'required' => false,
+                )
+            )
+            ->add(
+                'birthday_example3',
+                'birthday',
+                array(
+                    'widget' => 'single_text',
+                    'required' => false,
+                )
+            )
+            ->add(
+                'birthday_example4',
+                'birthday',
+                array(
+                    'widget' => 'single_text',
+                    'datepicker' => true,
+                    'required' => false,
+                )
+            )
+            ->add(
+                'save',
+                'submit',
+                array(
+                    'label' => 'Ok'
```

Comment:
Heeft het eigenlijk nut om hier een submit button te voorzien? Het gaat toch puur over de datepickers in het voorbeeld?